Firebase/Auth/Source/FirebaseAuthVersion.m

```diff
@@ -20,7 +20,7 @@
 #define STR(x) STR_EXPAND(x)
 #define STR_EXPAND(x) #x
 
-const double FirebaseAuthVersionNumber = FIRAuth_MINOR_VERSION;
+const double FirebaseAuthVersionNum = FIRAuth_MINOR_VERSION;
```

Comment:
Version number names tend not to be abbreviated in other projects. Is there any reason to change these?

Reply:
This change is to avoid conflicting with the default FirebaseAuthVersionNumber that gets generated in the umbrella header.